hamilton/io/utils.py

```diff
@@ -0,0 +1,19 @@
+import os
+from datetime import datetime
+from typing import Any, Dict
+
+
+def get_file_metadata(path: str) -> Dict[str, Any]:
+    """Gives metadata from loading a file.
+    This includes:
+    - the file size
+    - the file path
+    - the last modified time
+    - the current time
+    """
+    return {
+        "size": os.path.getsize(path),
+        "path": path,
+        "last_modified": os.path.getmtime(path),
+        "loaded_at": datetime.now().utcnow().timestamp(),
```

Comment:
`loaded_at` doesn't make sense? Shouldn't it be `touched_at` or `inspected_at` ?

Reply:
I spent like 3 years trying to figure out what the `L` in `ETL` stood for and now we've decided it doesn't mean what it always did 🤯 😄  But yeah, agreed.